src/pages/billing/NewBillingProjectWizard/StepWizard/Step.test.ts

```diff
@@ -22,9 +22,9 @@ describe('Step', () => {
   })
 
   it('sets aria-current to step when active', () => {
-    const renderResult = render(h(div, {}, [
-      Step({ isActive: true, children: [div({ children: ['activeStepChild'] })] }),
-      Step({ isActive: false, children: [div({ children: ['inactiveStepChild'] })] })
+    const renderResult = render(div([
+      h(Step, { isActive: true }, [div(['activeStepChild'])]),
+      h(Step, { isActive: false }, [div(['inactiveStepChild'])])
```

Comment:
This was the cause of the warning. If children is an array, React wants each child to have a unique key.